### DISCUSSION TITLE
How to suppress section title?

### DISCUSSION THREAD

**Discussion by mkvasnicka** [quarto-dev/quarto-cli#2044] [Q&A] [answered]
Hi!

I need to include references to chapters, sections, etc., in our book. If I use the following

```
# My First Chapter {#sec-first-chapter}

See @sec-first-chapter...
```

the result is "See Chapter 1". I want to get rid of the word "Chapter"; and in the same way, of "Section" etc. I thought I could use 

```
See [-@sec-first-chapter]...
```

but it doesn't work. (It works for figures and tables, though.) How can I omit these words?

Many thanks.
Michal

**Comment by mkvasnicka** [comment, marked as answer]
Sorry, my fault. Upgrading quarto from 0.9.442 to 1.0.38 did the trick. Apology.

**Comment by mcanouil** [comment]
You can also customise the prefix with `crossref-sec-prefix`, see <https://github.com/quarto-dev/quarto-cli/blob/main/src/resources/language/_language.yml>.
